### ISSUE TITLE
Expected date is off by one day

### ISSUE THREAD

**Issue by tedcook94** [tborychowski/courier#1]
The Expected field is off by a day on all of my deliveries.

You can see in the network response that "expected_delivery" for the "Rift S" tracking is "2021-05-17", but Courier displays "Expected" as "Sun May 16 2021". 

I suspect this has something to do with the Javascript Date object converting between UTC and local time.

![Capture](https://user-images.githubusercontent.com/33679293/118068195-918ef280-b367-11eb-9608-a2ba9a3af0c0.PNG)

**Comment by tborychowski** [comment]
Hi,

I didn't realise that anyone is actually using it! :-D

So the date parsing happens here:
https://github.com/tborychowski/courier/blob/53c0aa87904a05653baf1986b30bfd95120ba3fd/client/lib/store.js#L83-L84

If you get an incorrect offset - it may be caused by a different timezone?
Could you do a quick test for me, please?

paste these into the dev-tools console (1 by one) and let me know the output?

1. 
```js
new Date('2021-05-17').toDateString();
```
My output shows: "Mon May 17 2021"

2.
```js
new Date(2021, 4, 17).toDateString()
```
My output shows (the same): "Mon May 17 2021"


I assume yours are different, and that your timezone is UTC - 1, as `Date.parse` parses the date in the UTC format.
If so - I will correct my parsing with the 2nd method above (which I assume should work here).

Thanks!

**Comment by tedcook94** [comment]
Yes, that does seem to fix the issue (see my screenshot). My timezone is UTC-5 currently (-6 in the winter), so when it parses the date string it creates an object for 19:00 the day before.

Thanks for checking this out!

![Capture](https://user-images.githubusercontent.com/33679293/118132970-e3fefc00-b3c5-11eb-8a08-9f4a49c846c2.PNG)

**Comment by tborychowski** [comment]
I just released v0.0.8 (also to docker hub).
Please check it out and let me know if that fixes your issue.
(feel free to reopen if it wasn't fixed).

**Comment by tedcook94** [comment]
New Docker image fixed the issue for me. Thanks for fixing it!
